src/main/plugin/iso19139.ca.HNAP/schema-ident.xml

```diff
@@ -51,4 +51,15 @@
         </gmd:metadataStandardName>
 	    </elements>
 	</autodetect>
+  <filters xmlns:gco="http://www.isotc211.org/2005/gco">
+    <filter xpath=".//*[@gco:nilReason='withheld']"
+            ifNotOperation="editing">
+      <keepMarkedElement gco:nilReason="withheld"/>
+    </filter>
+    <filter
+      xpath=".//gmd:onLine[*/gmd:protocol/gco:CharacterString = 'WWW:DOWNLOAD-1.0-http--download']"
+      ifNotOperation="download"/>
+    <filter xpath="*//gmd:onLine[starts-with(*/gmd:protocol/gco:CharacterString, 'OGC:WMS')]"
```

Comment:
Would we also need to include protocols that start with "ESRI REST:"?  I'm guessing that this bug exists on iso19139

Reply:
I think you are right, and probably to check also `OGC:WFS`